db/migrate/20241220001400_rename_user_bookmarks.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUserBookmarks < ActiveRecord::Migration[7.2]
+  def change
+    rename_table :user_bookmarks, :bookmarks
```

Comment:
(Can probably squish this with the above migration)

Reply:
Just delete the first one and update this one, or?